include/bout/mesh.hxx

```diff
@@ -581,6 +583,14 @@ class Mesh {
   /// Global index excludes boundary cells, local index includes boundary or guard cells.
   virtual int getGlobalXIndexNoBoundaries(int xlocal) const = 0;
 
+  /// Returns a local X index given a global index.
+  /// Global index includes boundary cells, local index includes boundary or guard cells.
+  virtual int getLocalXIndex(int xlocal) const = 0;
```

Comment:
I think here and in the other `getLocal*` methods, the argument should be `xglobal`, etc. as the argument is a global-index value.
```suggestion
  virtual int getLocalXIndex(int xglobal) const = 0;
```

Reply:
Thanks! I did change it in some places, but evidently didn't follow it through